src/clojure/nrepl/misc.clj

```diff
@@ -60,3 +60,8 @@
         (require (symbol (namespace sym)))
         (resolve sym)
         (catch Exception _))))
+
+(defn keyworded-set
```

Comment:
Such helpers should be kept private and in the namespaces that need them. The misc ns exposes functions useful to client writes (the name is a bit confusing).

Reply:
I suspect they can be all moved into `core-test`, where it's mostly used.

Reply:
Removed most usages of this. Then rolled into `clean-response` in `core-test`